src/agent/azureFunctionsLoader.ts

```diff
@@ -1,14 +1,13 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { Resource } from "@opentelemetry/resources";
 import { AgentLoader } from "./agentLoader";
 import { DiagnosticLogger } from "./diagnostics/diagnosticLogger";
 import { StatusLogger } from "./diagnostics/statusLogger";
 import { AzureFunctionsWriter } from "./diagnostics/writers/azureFunctionsWriter";
 import { AgentResourceProviderType, AZURE_MONITOR_AGENT_PREFIX } from "./types";
-import { Attributes } from "@opentelemetry/api";
-import { SemanticResourceAttributes } from "@opentelemetry/semantic-conventions";
+import { detectResourcesSync } from '@opentelemetry/resources';
+import { azureFunctionsDetector } from '@opentelemetry/resource-detector-azure/build/src/detectors/AzureFunctionsDetector';
```

Comment:
Excuse the strange import path here. Functions detector isn't exported from the index in the resource detector. Addressed here: https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1800